### PR TITLE
Feature/22 sidebar styling

### PR DESCRIPTION
**Pull request by sbrody**
Adds GOV.UK styling to the sidebar component. Applies new styles using https://www.gov.uk/growing-your-business as an example. The example uses a class not available to this project so the styles are brought over individually. The focus is on borders, font sizes and ensuring fields do not extend beyond the container width.

Note: the 'category-2' widget is no longer available, so it was not possible to extensively test this component.

image

### CODE REVIEW

**Review by matthewpassmore — CHANGES_REQUESTED** [review]
Can we look at maintaining consistent spacing between child items across all widgets? Currently, it's looking like this for me:

Screenshot 2024-04-10 at 17 45 27

Ideally, the vertical rhythm would be the same for each section (e.g. _Contributers_, _Follow us_). Using [Growing your business](https://www.gov.uk/growing-your-business) as a reference, this would be along the lines of:

- `15px` top margin on all elements
- `10px` space beneath section headings
- `30px` space at foot of section [DONE]

**Comment by sbrody** [comment]
> Can we look at maintaining consistent spacing between child items across all widgets? Currently, it's looking like this for me:
> Screenshot 2024-04-10 at 17 45 27
> 
> Ideally, the vertical rhythm would be the same for each section (e.g. _Contributers_, _Follow us_). Using [Growing your business](https://www.gov.uk/growing-your-business) as a reference, this would be along the lines of:
> 
>     * `15px` top margin on all elements
> 
>     * `10px` space beneath section headings
> 
>     * `30px` space at foot of section [DONE]

I've pushed up some changes which should address this

File: assets/css/components/sidebar.scss [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
+	p,
+	.wp-block-heading,
+	h3,
+	select {
+		font-size: 1rem;
+	}
+
+	.widget-inner > * > *,
+	.wp-block-latest-comments__comment {
+		margin-bottom: $spacingSmall;
```
Comment (matthewpassmore) [inline assets/css/components/sidebar.scss:28]: This margin equates to `5px`, whereas the spacing on GOV.UK is `15px`. The govuk spacing function can be utilised to maintain consistency e.g. `margin-bottom: govuk-spacing(3)`

**Review by matthewpassmore — CHANGES_REQUESTED** [review]

File: assets/css/components/sidebar.scss (hunk 1 above)
Comment (sbrody) [reply assets/css/components/sidebar.scss]: Ok - have adjusted to increase to 15px - it looks a bit loose to me but should be consistent with gov.uk

**Review by matthewpassmore — APPROVED** [review]
All good 👍

**Pull request merged**